yaml/com/ibm/License/LicenseManager.interface.yaml

```diff
@@ -0,0 +1,41 @@
+description: >
+      This Method accepts license key and forwards to host
+properties:
+     - name: LicenseString
+       type: string
+       description: >
+         License string of 34 characters.
+     - name: LicenseActivationStatus
+       type: enum[self.Status]
+       default: Pending
+       description: >
+         License Activation status
+enumerations:
+     - name: Status
+       description: >
+         Status enumerations
+       values:
+        - name: InvalidLicense
+          description: >
+            Indicates that given license is not valid
+        - name: Activated
+          description: >
+            Indicates that given license is activated
+        - name: Pending
+          description: >
+            Indicates that given license activation is yet to happen
+        - name: ActivationFailed
+          description: >
+            Indicates that given license activation failed due to some firmware issue
```

Comment:
This would always be a firmware issue? 
             Indicates that given license activation failed due to some a firmware or other error ?

Reply:
done